package.json

```diff
@@ -90,5 +96,19 @@
     "ts-patch": "^2.1.0",
     "typescript": "^4.9.3",
     "typescript-transform-paths": "^3.4.6"
+  },
+  "dependenciesMeta": {
+    "buildcheck@0.0.6": {
+      "unplugged": true
+    }
+  },
```

Comment:
Is this left on purpose?

Reply:
Yes, I was getting [this](https://github.com/mscdex/buildcheck/issues/1#issuecomment-1119903300) error until I put this in the package.json. I can try removing and seeing if it is still an issue as I recently dist upgraded to Ubuntu 22 from 20.

Reply:
To confirm, I removed this and everything still works.